src/Body/EmojiGrid/style.css

```
@@ -0,0 +1,3 @@
+.grid-Emoji{
```

```suggestion
.gridEmoji{
```
Be consistent with you naming style